code/modules/mob/living/simple_animal/bot/construction.dm

```diff
@@ -166,7 +166,7 @@
 						return
 					newname = "redtag ED-209 assembly"
 				if("")
-					if(!istype(W, /obj/item/gun/energy/e_gun/advtaser))
+					if(!istype(W, /obj/item/gun/energy/e_gun/))
```

Comment:
ED209 needs to have it's projectiles updated to reflect that it's using a normal egun now, ie disablers & lethal lasers.

Reply:
As well as a normal taser option added to it's interface since this would also allow building it with the captain's gun, and the HOS's.

Reply:
hm.... this would also allow you to build ED209's with dragnets.